web/client/components/widgets/builder/wizard/LegendWizard.jsx

```diff
@@ -19,6 +21,14 @@ const Wizard = wizardHandlers(require('../../../misc/wizard/WizardContainer'));
 const StepHeader = require('../../../misc/wizard/StepHeader');
 const Message = require('../../../I18N/Message');
 const emptyLegendState = require('../../enhancers/emptyLegendState');
+const {currentLocaleSelector} = require('../../../../selectors/locale');
+
+const localeSelector = createSelector(
```

Comment:
Could you please connect the state at plugin level (`plugins/widgetBuilder/LegendBuilder.jsx`).
We use to connect component only at plugin level (we will define better this policy in next tech debt

Reply:
done